e2e-playwright/dashboard-cujs/dashboard-navigation.spec.ts

```diff
@@ -0,0 +1,243 @@
+import { test, expect } from '@grafana/plugin-e2e';
+
+import scopesDashboard1 from '../dashboards/scopes-cujs/db1.json';
+import scopesDashboard2 from '../dashboards/scopes-cujs/db2.json';
+
+import {
+  applyScopes,
+  expandScopesSelection,
+  openScopesSelector,
+  scopeSelectRequest,
+  searchScopes,
+  selectScope,
+  TestScope,
+} from './scopeUtils';
+import { testScopes } from './scopes';
+
+test.use({
+  featureToggles: {
+    scopeFilters: true,
+    groupByVariable: true,
+    reloadDashboardsOnParamsChange: true,
+  },
+});
+
+export const FIRST_DASHBOARD = process.env.USE_LIVE_DATA
+  ? '30c27714f467e4fb6445c7a957fd15ed/mimir-overview-resources'
+  : 'scopes-dashboard-1';
+export const SECOND_DASHBOARD = process.env.USE_LIVE_DATA
+  ? '06b1705e8960c1ad77caf7f3eba3caba/mimir-writes'
+  : 'scopes-dashboard-2';
+
+test.describe(
+  'Dashboard navigation',
+  {
+    tag: ['@dashboard-cujs'],
+  },
+  () => {
+    let dashboardUIDs: string[] = [];
+
+    test.beforeAll(async ({ request }) => {
+      // Import the test dashboard
+      for (const dashboard of [scopesDashboard1, scopesDashboard2]) {
+        let response = await request.post('/api/dashboards/import', {
```

Comment:
Ideally instead of relying on direct interaction with API paths, we should use `getDashboardAPI` that would handle correct API version retrieval based on the provided feature toggles.

Reply:
Indeed that would be ideal, I've tried doing that but cannot get to import the dashboardAPI service properly.